src/main/java/org/checkerframework/specimin/UnsolvedClass.java

```diff
@@ -50,6 +53,21 @@ public UnsolvedClass(@ClassGetSimpleName String className, String packageName) {
     this.classFields = new LinkedHashSet<>();
   }
 
+  /**
+   * Create an instance of UnsolvedClass
+   *
+   * @param className the name of the class
+   * @param packageName the name of the package
+   */
+  public UnsolvedClass(
+      @ClassGetSimpleName String className, String packageName, boolean exceptionType) {
```

Comment:
Rather than updating all call sites of this constructor, it would be better to add a second constructor that has the same signature as the constructor before your PR, and which calls this constructor with `exceptionType` set to false.

Reply:
Current code has two constructors. 
```
public UnsolvedClass( @ClassGetSimpleName String className, String packageName)
public UnsolvedClass(@ClassGetSimpleName String className, String packageName, boolean exceptionType)
```
The second one is called where the `exceptionType` is `true`. In all other cases the first one is called. The default value of the `isExceptionType` is set to `false`;

I think you suggested this, am I right?

Reply:
Partially. The problem in the current implementation is actually the other (unchanged) constructor: it should call this constructor explicitly rather than re-implementing the assignment of fields. In other words, my suggestion is already partially (but not completely) implemented.

Reply:
public UnsolvedClass(
      @ClassGetSimpleName String className, String packageName, boolean exceptionType) {
    this(className, packageName);
    this.isExceptionType = exceptionType;
  }

Reply:
The dependency should go the other way: the other constructor's implementation should be:
```
public UnsolvedClass(@ClassGetSimpleName String className, String packageName) {
  this(className, packageName, false);
}
```

Reply:
Thanks Professor. updated accordingly.